Decred Wallet/Extensions/UIColor.swift

```diff
@@ -28,7 +28,12 @@ extension UIColor {
         static let thinGray = UIColor.init(hex: "#a4abb1")
         static let darkGray = UIColor.init(hex: "#4e5f70")
         static let transparentThinGray = UIColor.init(hex: "#a4abb1", alpha: 0.3)
-        static let darkBlue = UIColor.init(hex: "#091440")
+
+        static let darkBlue = UIColor(hex: "#3d5873")
```

Comment:
I'm not comfortable with this color replacement. The implication is that other views using the previous hex value would now use this new hex value. This new hex value should be a new color variable, perhaps `darkerBlue`. Retain the original `darkBlue` definition.